keylime/utils.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/python3
```

Comment:
Pretty nitpicky, but maybe rename this file to user_utils. In general, I find things just named "utils" leads to poor code organization in the future as people just dump things in there without really thinking about where they should really belong.

Reply:
Renamed.